crates/matrix-sdk-crypto-nodejs/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 The Matrix.org Foundation C.I.C.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod attachments;
+mod device;
+mod machine;
+mod models;
+mod request;
+mod responses;
```

Comment:
This is because my IDE was complaining that it couldn't determine types of "unused" modules. I think napi-rs also requires the things to be referenced like this, so might be why it's not complaining? 

Is the solution to just stick `pub` in front of these?

Reply:
You could do that or you could add exports for the public types, for example: `pub use machine::OlmMachine`.